gateway/src/main/scala/com/improving/app/gateway/domain/common/util.scala

```diff
@@ -21,4 +28,62 @@ object util {
       config.getInt(s"services.$serviceName.port")
     )
   }
+
+  def genEstablishTenantReqs(
```

Comment:
This really looks more like demo code, curious why it needed to move to the main part of the code base.

Reply:
Just refactoring to make it cleaner. It's where I decided to put it, but now we are moving all this to Gatling service, so it's no biggie